thirdparty/getJson.py

```diff
@@ -16,12 +16,12 @@ def getJson(install, prefix, par):
     osStr = (platform.system())
     cwd = os.getcwd()
     folder = cwd + "/json"
-    url = "https://github.com/nlohmann/json.git  "
+    url = "https://github.com/nlohmann/json.git"
+    json_branch_or_tag = "develop"
```

Comment:
Also, the previous version of the script to install nlohmann/json didn't explicitly specify a branch or tag or commit-id to pull, so it just pulled the default branch (`develop`). i have made this explicit in this PR. However, I don't know whether or not `develop` is stable, so if you think that it is appropriate you can change this to some release tag (e.g. `v3.11.3`)

```suggestion
    json_branch_or_tag = "v3.11.3"
```